gulp-tasks/workbox/workbox-generate-contributors.js

```diff
@@ -1,9 +1,11 @@
+'use strict';
```

Comment:
Add the JSDoc description to this file and include at least `@fileoverview` and `@author`

Reply:
Done.